src/value/borrowed.rs

```diff
@@ -195,10 +268,33 @@ impl<'val> Struct for BorrowedStruct<'val> {
     }
 }
 
+impl<'val> PartialEq for BorrowedStruct<'val> {
+    fn eq(&self, other: &Self) -> bool {
+        // check if both the text_fields have same (field_name,value) pairs
+        self.text_fields.iter().all(|(key, value)| {
+            value.iter().all(|(_my_s, my_v)| {
+                other
+                    .get_all(key)
+                    .find(|other_v| my_v == *other_v)
+                    .is_some()
```

Comment:
Codecov is complaining that L275-278 and L280 is not tested, please confirm (could be a misattribution, but worth checking).